src/Routes/RosDetailsPage/RosDetailsPage.js

```diff
@@ -1,15 +1,138 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
 
+import { Link } from 'react-router-dom';
+import { Grid, GridItem, Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
 import { Main } from '@redhat-cloud-services/frontend-components/Main';
-import { Unavailable } from '@redhat-cloud-services/frontend-components/Unavailable';
-
-const RosDetailsPage = () => {
-    return (
-        <Main>
-            <Unavailable/>
-        </Main>
-    );
+import { PageHeader } from '@redhat-cloud-services/frontend-components/PageHeader';
+import { InventoryDetailHead, AppInfo, DetailWrapper } from '@redhat-cloud-services/frontend-components/Inventory';
+import { register } from '../../store';
+import { entityDetailReducer } from '../../store/entityDetailReducer';
+// import { fetchRosSystemDetailsAction } from '../../store/Actions/Actions';
+import './ros-details-page.scss';
+import { ExpandedRow } from '../../Components/RosTable/ExpandedRow';
+import { ROS_API_ROOT, SYSTEMS_API_ROOT } from '../../constants';
+import RecommendationRating from '../../Components/RecommendationRating/RecommendationRating';
+
+class RosDetailsPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            inventoryId: this.props.match.params.inventoryId
+        };
+    }
+
+    async componentDidMount() {
+        insights.chrome?.hideGlobalFilter?.(true);
+        insights.chrome.appAction('system-detail');
+        await window.insights.chrome.auth.getUser();
+        await this.fetchRosSystemDetails();
+    }
+
+    async fetchRosSystemDetails() {
+        let url = new URL(ROS_API_ROOT + SYSTEMS_API_ROOT + `/${this.state.inventoryId}`,  window.location.origin);
```

Comment:
@karelhala, 
One query - I tried to make this change using action & reducer so that it will add ROS system info in store. But it wasn't giving call to reducer function for type `FETCH_ROS_SYSTEM_DETAIL_FULFILLED`. 
Apart from [line-81](https://github.com/RedHatInsights/ros-frontend/pull/24/files#diff-fcf73cad52850091e1789c058644cf5a36de60d32f3014517b98385dac732221R81) change, do I need to make any extra change to get it work?